Assets/VRTK/Scripts/VRTK_InteractGrab.cs

```diff
@@ -37,6 +37,8 @@ public class VRTK_InteractGrab : MonoBehaviour
         public float grabPrecognition = 0f;
         [Tooltip("An amount to multiply the velocity of any objects being thrown. This can be useful when scaling up the `[CameraRig]` to simulate being able to throw items further.")]
         public float throwMultiplier = 1f;
+        [Tooltip("Automatically scale the throw velocity up and down to match the scale of the `[CameraRig]`.")]
+        public bool throwScalesWithPlayer = true;
```

Comment:
I'm wondering rather than just having a bool to do this on `velocity` we have 2 enum drop downs that provide options for both velocity and angular velocity so you can cater to whatever need you want.

Alternatively, we could look at doing this without additional parameters? You inherit the script and override the method? Or composition that you add a "throw strategy" script  into the mix?

what do you think about any of that?